src/lesson10/exceprions/userValidationService/UserValidator.java

```diff
@@ -0,0 +1,36 @@
+package lesson10.exceprions.userValidationService;
+
+public class UserValidator {
+
+    private void checkingValueForNull() {
+        throw new UserValidationException("The value cannot be null");
+    }
+
+    private void validateUserFirstName(User user) {
+        if (user.getFirstName() == null) checkingValueForNull();
```

Comment:
Move this if to the `checkingValueForNull` and call it `assertUserNotNull`

Reply:
I dit it